go.mod

```diff
@@ -2,15 +2,17 @@ module go.elastic.co/apm
 
 require (
 	github.com/armon/go-radix v1.0.0
-	github.com/davecgh/go-spew v1.1.1 // indirect
```

Comment:
There are a lot of go.mod changes unrelated to apmnegroni. Can you please run `go mod tidy` to revert them? There's a script to list the modules which you can run from the repo root dir to make this easier:

```sh
for dir in $(./scripts/moduledirs.sh); do (cd $dir && go mod tidy -v); done
```